### PR TITLE
[ET-981] Add confirmation dialog for bulk deletion of attendees.

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ET-981]

Screencast: https://d.pr/v/kq1v2m

[ET-981]: https://theeventscalendar.atlassian.net/browse/ET-981

### CODE REVIEW

File: src/resources/js/tickets-attendees.js [hunk 1]
```diff
@@ -63,6 +63,21 @@ var tribe_event_tickets_attendees = tribe_event_tickets_attendees || {};
...
 
+		$( '.attendees-admin-form' ).on( 'submit', function ( e ) {
+
+			// If not the delete action, return.
+			if ( 'delete_attendee' !== $( '#bulk-action-selector-top' ).val() ) {
+				return;
+			}
+
+			// If no attendee was selected, bail out.
+			if ( ! $( this ).serialize().includes( '&attendee' ) )  {
```
Comment (paulmskim) [inline src/resources/js/tickets-attendees.js:74]: Do we still support IE11? If we do, the `.includes()` function will not work on IE11. Instead, we'll want to use `.indexOf()`.

If we don't care about IE11, then disregard this comment ;)

**Review by paulmskim — COMMENTED** [review]
a couple suggestions

File: src/resources/js/tickets-attendees.js (hunk 1 above)
Comment (juanfra) [reply src/resources/js/tickets-attendees.js:74]: We're no longer supporting IE11 https://docs.theeventscalendar.com/wiki/view/Crossbrowser_Testing

**Review by juanfra — APPROVED** [review]

**Comment by juanfra** [comment]
Let's address Paul's suggestion and we're good to go 🎉

File: src/admin-views/attendees.php [hunk 2]
```diff
@@ -162,7 +162,7 @@
 	</div>
 	<?php do_action( 'tribe_events_tickets_attendees_event_summary_table_after', $event_id ); ?>
 
-	<form id="topics-filter" class="topics-filter" method="post">
+	<form id="attendees-admin-form" class="topics-filter attendees-admin-form" method="post">
```
Comment (sc0ttkclark) [inline src/admin-views/attendees.php:165]: If we're introducing ids/classes, we should namespace them at minimal

File: src/resources/js/tickets-attendees.js [hunk 3]
```diff
@@ -63,6 +63,21 @@ var tribe_event_tickets_attendees = tribe_event_tickets_attendees || {};
 			return confirm( Attendees.confirmation );
 		});
 
+		$( '.attendees-admin-form' ).on( 'submit', function ( e ) {
```
Comment (sc0ttkclark) [inline src/resources/js/tickets-attendees.js:66]: Namespace class

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs class/id namespacing

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**